### PR TITLE
Resolve #90

### LINKED ISSUE #90: Jar Files in lib Folder Not Recognized
### Description:
Jar files within the `lib` folder are not recognized using the sbt-projectmatrix plugin.  
about lib folder: https://www.scala-sbt.org/1.x/docs/Library-Dependencies.html#Unmanaged+dependencies

### Reproduction:
A reproduction project can be found here: [Reproduction Project](https://github.com/kijuky/sbt-projectmatrix-lib)

### Steps to Reproduce:
1. Clone the reproduction project.
2. Follow the steps outlined in the project's README to reproduce the issue.

### Expected Behavior:
Jar files within the `lib` folder should be recognized and utilized by sbt-projectmatrix.

### Actual Behavior:
The jar files are not recognized by sbt-projectmatrix.

### Additional Information:
- This issue affects the functionality and usability of sbt-projectmatrix.
- Further investigation and resolution are required.

### PR DESCRIPTION
**Pull request by kijuky**
Resolve #90

### CODE REVIEW

File: src/sbt-test/projectMatrix/lib/project/build.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+sbt.version=1.9.9
```
Comment (xuwei-k) [inline src/sbt-test/projectMatrix/lib/project/build.properties]: don't add `build.properties` in scripted test

Comment (kijuky) [reply src/sbt-test/projectMatrix/lib/project/build.properties]: Thank you! Delete this file. :hand:

Comment (kijuky) [reply src/sbt-test/projectMatrix/lib/project/build.properties]: - Delete build.properties (use sbt 1.2.8)
- Changed the combination of Scala versions to be cross-compiled from (2.11, 3.3) to (2.12, 2.13)

**Review by eed3si9n — APPROVED** [review]
Thanks!

**Pull request merged**